web/src/app/queries/queries.js

```diff
@@ -5,10 +5,29 @@ import API from 'API';
 
 export const getProducts = Query({
   id: 'getProducts',
-  params: { sortBy: t.Any, ascending: t.maybe(t.Boolean) },
+  params: {
+    exchanges: t.Any,
+    base: t.Any,
+    quote: t.Any,
+    sortBy: t.Any,
+    ascending: t.maybe(t.Boolean)
+  },
   cacheStrategy: new Expire(15000),
   returnType: t.Any,
-  fetch: ({ sortBy = 'Exchange', ascending = true }) =>
-    API.productsController_read({ sortBy, ascending })
+  fetch: ({
+    exchanges = ['Bitfinex', 'Bitstamp', 'Kraken'],
+    base = undefined,
+    quote = undefined,
+    sortBy = 'Exchange',
+    ascending = true
+  }) => {
+    return API.productsController_read({
+      exchanges: JSON.stringify(exchanges), // doing this to prevent wrong serialization
```

Comment:
@giogonzo oggi io e @MatteoTurri abbiamo fatto questo hack perchè metarpheus non genera cose per la serializzazione di quello che `wiro` il backend (`wiro`) si aspetta come lista. È un bug tracciato che tu sappia?

Reply:
right, argomento delicato ma non tracciato: così come le risposte da api (e da wiro) vengono deserializzzate, i parametri in input dovrebbero essere serializzati prima di essere inviati.
Ho aperto https://github.com/buildo/metarpheus-io-ts/issues/30